CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 4.14.0
+
+- Allow use of multiple GA customDimensionIndex.
+- Configurable duration (in days) for AB Test cookie.
+- Add assignment of jQuery explicitly so it's consistent with other modules.
```

Comment:
Probably worth mentioning / linking to [the PR](https://github.com/alphagov/govuk_frontend_toolkit/pull/290) for this one, if someone wants more context.